docs/cli.rst

```diff
@@ -15,13 +15,81 @@ Generic commands
-    {linux,mac,windows}   host os name
-    {desktop,winrt,android,ios} target sdk
+List command
+------------------
+
+.. program::  aqt
+
+.. option:: list [-h | --help] [--extension <extension>] [--filter-minor <Qt minor version>]
+                [--modules (<Qt version> | latest) | --extensions (<Qt version> | latest) |
+                 --arch (<Qt version> | latest) | --latest-version | --tool <tool name>]
+                <category> <target OS> [<target variant>]
+
+    list available tools, versions of Qt, targets, extensions, modules, and architectures.
+
+.. describe:: category
+
+    tools, qt5 or qt6
+
+.. describe:: target OS (aka host in code/help text)
+
+    linux, windows or mac
+
+.. describe:: target variant (aka target in code/help text)
+
+    desktop, winrt, ios or android. When omitted, the command prints all the targets available for a host OS.
+
+.. option:: --help, -h
+
+    Display help text
+
+.. option:: --extension {wasm,src_doc_examples,preview,wasm_preview,x86_64,x86,armv7,arm64_v8a}
+
+    Extension of packages to list.
+    Use the `--extensions` flag to list all relevant options for a host/target.
+    Incompatible with the `--extensions` flag, but may be combined with any other flag.
+
+.. option:: --extensions (<Qt version> | latest)
+
+    Qt version in the format of "5.X.Y", or the keyword `latest`.
+    When set, this prints all valid arguments for the `--extension` flag for
+    Qt 5.X.Y, or the latest version of Qt if `latest` is specified.
+    Incompatible with the `--extension` flag.
+
+.. option:: --filter-minor <Qt minor version>
+
+    Print versions of Qt that have a particular minor version.
+    For example, `aqt list qt5 windows desktop --filter-minor 12` would print
+    all versions of Qt for Windows Desktop beginning with 5.12.
+    May be combined with any other flag to filter the output of that flag.
+
+.. option:: --modules (<Qt version> | latest)
+
+    Qt version in the format of "5.X.Y". When set, this lists all the modules
+    available for Qt 5.X.Y with a host/target/extension, or the latest version
+    of Qt if `latest` is specified.
+
+.. option:: --arch (<Qt version> | latest)
+
+    Qt version in the format of "5.X.Y". When set, this prints all architectures
+    available for Qt 5.X.Y with a host/target/extension, or the latest version
+    of Qt if `latest` is specified.
+
+.. option:: --latest-version
+
+    Print only the newest version available
+    May be combined with the `--extension` and/or `--filter-minor` flags.
+
+.. option:: --tool <tool name>
+
+    The name of a tool. Use `aqt list tools <host> <target>` to see accepted values.
+    This flag only works with the 'tools' category, and may noy be combined with
+    any other flags.
+    When set, this prints all 'tool variant names' available.
+
+    The output of this command is meant to be used with the `aqt tool` command:
+    See the :ref:`Tool installation commands` below.
```

Comment:
CI logs make it look like this will be a broken link. There's probably a setting somewhere to turn this feature on; I could try to figure it out if you want this link to work. Otherwise, I can just take the link out; I don't think the link is all that helpful.

Reply:
We can add `extensions = ['sphinx.ext.autosectionlabel']` in docs/conf.py

Reply:
I've pushed it. Please see.